src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions_Shared.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
all the files intended to be shared end with the `_Shared.cs` suffix.  I believe these should all be pure splits into partial types, and pure moves of the code.